Assets/Scripts/Controllers/TileSpriteController.cs

```diff
@@ -75,8 +80,20 @@ protected override void OnChanged(Tile tile)
             Debug.ULogErrorChannel("TileSpriteController", "tileGameObjectMap's returned GameObject is null -- did you forget to add the tile to the dictionary? Or maybe forget to unregister a callback?");
             return;
         }
-        
-        tile_go.GetComponent<SpriteRenderer>().sprite = SpriteManager.current.GetSprite("Tile", tile.Type.Name);
+
+        //TODO Evaluate this criteria and naming schema!
```

Comment:
Changing it if we don't like it is the default if you don't put in the comment. You might even get better suggestions on it if you put it out to people rather than a TODO comment.

By putting it in the code you are requiring someone to make a PR to either remove the comment if we agree on it or to remove it and change the names.